src/Reflex/Dom/Contrib/Widgets/ButtonGroup.hs

```diff
@@ -167,15 +167,15 @@ radioGroup dynName dynEntryList cfg = do
      <> bool mempty ("checked" =: "checked") chkd
     handleOne _ dynV dynChecked = do
 
-      el "tr" $ do
+      el "tr" $ el "td" $ el "label" $  do
```

Comment:
Thanks for this PR.  Love the idea!  My main concern is that this switches us from using a single `<td>` to using two, which might break people who are already using this.  Is there a way you can accomplish the same thing and maintain the existing `<td>` structure?